### ISSUE TITLE
extension _yaml is not built

### ISSUE THREAD

**Issue by dfroger** [yaml/pyyaml#179]
Trying to build the extension `_yaml` for `pyyaml 4.1` on `debian 8`:

    python setup.py build_ext

Fails with:

```
/home/dfroger/.pyenv/versions/3.6.2/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'python_requires'
  warnings.warn(msg)
running build_ext
building '_yaml' extension
creating build
creating build/temp.linux-x86_64-3.6
creating build/temp.linux-x86_64-3.6/ext
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -Wstrict-prototypes -fPIC -I/home/dfroger/.pyenv/versions/3.6.2/include/python3.6m -c ext/_yaml.c -o build/temp.linux-x86_64-3.6/ext/_yaml.o
gcc: error: ext/_yaml.c: No such file or directory
gcc: fatal error: no input files
compilation terminated.
Error compiling module, falling back to pure Python
```

Note that directory `build/temp.linux-x86_64-3.6/ext` is empty.

**Comment by dfroger** [comment]
When `cython` is installed, building the extension works.

**Comment by dfroger** [comment]
The problem is that in https://pypi.org/simple/pyyaml/ `PyYAML-3.12.tar.gz` contains `ext/_yaml.c`, but `PyYAML-4.1.tar.gz` do not, so `cython` is required for `4.1` to build the extension.

@sigmavirus24 could you make a `4.1` tarball that contains `ext/_yaml.c`, or would you accept a PR on that?

**Comment by ingydotnet** [comment]
I'll look into this as soon as our issues around building wheels are resolved.

**Comment by ingydotnet** [comment]
FYI PyYAML release 4.1 was retracted but this has already been fixed in 4.2b1 by applying https://github.com/yaml/pyyaml/pull/188
